### PR TITLE
Manga Pro cover and chapter list bugs fixed

### LINKED ISSUE #11809: Manga Pro Z (AR): Cover image gets deleted after first manga details update
### Source information

Manga Pro Z (AR) 1.2.15

### Steps to reproduce

1. Browse the source
2. Notice the cover is loaded correctly first time
3. go inside any manga and update details
4. the cover image will get deleted
5. go back and notice the cover is no longer shown when viewing the manga from browse/library

### Expected behavior

Cover should get updated

### Actual behavior

Cover gets deleted

### Tachiyomi version

0.13.4

### Android version

Android 9

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #11881: Manga Pro Z: Pagelist returns Error 404
### Source information

Manga Pro Z [ar] 1.2.15

### Steps to reproduce

open manga "I Am Carrying Gold From the Post-Apocalyptic World"
try to download or browse latest chapter

### Expected behavior

Should work normally like other chapters in the same manga

### Actual behavior

Throwing Error 404 and no pages get downloaded at all. I checked webview and the chapter is working normally. But is protected first by an ad URL shortner.

### Tachiyomi version

j2k 1.5.2

### Android version

Android 9

### Other details

I only found the issue at latest chapter, the earlier ones are okay.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Saud-97**
Fixes #11881 and fixes #11809

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/overrides/wpmangastream/mangaproz/src/MangaPro.kt [hunk 1]
```diff
@@ -2,15 +2,47 @@ package eu.kanade.tachiyomi.extension.ar.mangaproz
...
+import eu.kanade.tachiyomi.source.model.SManga
 import okhttp3.OkHttpClient
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
 import java.util.concurrent.TimeUnit
 
 class MangaPro : WPMangaStream("Manga Pro", "https://mangaprotm.com", "ar") {
-    override val id: Long = 964048798769065340
+	override val id: Long = 964048798769065340
```
Comment (AntsyLich) [inline multisrc/overrides/wpmangastream/mangaproz/src/MangaPro.kt:14]: that's a 3 space indent...

**Comment by AntsyLich** [comment]
About the cover thing should just pr to wpmangastream

File: multisrc/overrides/wpmangastream/mangaproz/src/MangaPro.kt (hunk 1 above)
Comment (Saud-97) [reply multisrc/overrides/wpmangastream/mangaproz/src/MangaPro.kt:14]: Why not use default tab indentation?

**Comment by Saud-97** [comment]
> About the cover thing should just pr to wpmangastream

I have no idea if it may cause issues with other extensions. Are you sure it won't introduce new bugs?

File: multisrc/overrides/wpmangastream/mangaproz/src/MangaPro.kt (hunk 1 above)
Comment (AntsyLich) [reply multisrc/overrides/wpmangastream/mangaproz/src/MangaPro.kt:14]: default is 4 tho

Comment (Saud-97) [reply multisrc/overrides/wpmangastream/mangaproz/src/MangaPro.kt:14]: This is probably a github bug not from my PR. I have it as 4 and this is just a single tab.

**Comment by alessandrojean** [comment]
Same as the other PR, use spaces instead of tabulation to ident.

**Comment by alessandrojean** [comment]
> Are you sure it won't introduce new bugs?

Could be added later if more websites starts using it.

**Pull request merged**